examples/go/api_test.go

```diff
@@ -1,11 +1,12 @@
 package twilio
 
 import (
-	"github.com/stretchr/testify/assert"
-	openapi "github.com/twilio/twilio-go/twilio/rest/oai"
 	"os"
 	"testing"
 	"time"
+
+	"github.com/stretchr/testify/assert"
+	openapi "github.com/twilio/twilio-go/twilio/rest/oai"
```

Comment:
This location seems to not exist. The tests run great via `make docker-test`, but when I try `go get github.com/twilio/twilio-go/twilio/rest/oai` locally, I get this error: `go get github.com/twilio/twilio-go/twilio/rest/oai: no matching versions for query "upgrade"`

Reply:
This is a local error that is related to your Go set up. Currently it's probably using your local gopath to try to run these tests. It needs to use the go modules in the example/go directory to get `github.com/twilio/twilio-go/twilio/rest/oai`

Reply:
Can we please document this somewhere, thanks!

Reply:
Sure! Any suggestions on where we might want to document this?

Reply:
Maybe our Wiki is sufficient for now?

This may come up when one of our team later tries to do local testing, so maybe in the Local Setup area.